runtime/tests/jniarg/cdefs.c

```diff
@@ -153,5 +154,11 @@ void JNICALL Java_JniArgTests_summary( JNIEnv *p_env, jobject p_this )
 	} else {
 		j9tty_printf( PORTLIB, "JNI ARG Tests PASSED\n" );
 	}
+
+	if ((argFailures + retValFailures) > 0) {
+		rc = (jint)1;
```

Comment:
The cast is not necessary here (or a cast is needed in the declaration).

Reply:
Agreed, I would remove it rather than add another.

Reply:
Fixed in ab713ca (see [force push](https://github.com/eclipse/openj9/compare/5f938ef88bb3f8d33239c8a3c05657370baa457e..ab713cac6f91859325f0e62136936b0bb7ddbec8)).